spyder/plugins/completion/languageserver/transport/main.py

```diff
@@ -31,6 +32,8 @@
 
 logger = logging.getLogger(__name__)
 
+PARENT_PROCESS_WATCH_INTERVAL = 10  # 10 s
```

Comment:
Let's reduce this to 3 secs to try to be sure a previous PyLS is not present when a new Spyder session is started.